src/newrelic/app_test.go

```diff
@@ -59,10 +60,14 @@ func TestConnectPayloadInternal(t *testing.T) {
 	}
 
 	expected.Util.Hostname = info.Hostname
+	utilization.OverrideDockerId(expected.Util, info.DockerId)
```

Comment:
I don't think this is necessary or a good idea to have the expected depend on a function.

Reply:
It is necessary- I can't assign the `vendor` struct to `expected.Util` from outside the utilization package. If I can't call this function to set the docker id in the `vendor` struct, I have to remove this test entirely.

Reply:
Maybe this is redundant then as I don't think setting the expected value by calling `OverrideDockerId` is what we want.  
My understanding is this test is verifying `ConnectPayloadInternal()` will get the proper Docker Id when a value is present in the `info` struct.  Is that what the `TestConnectPayloadInternalDocker` test does?

Reply:
> My understanding is this test is verifying ConnectPayloadInternal() will get the proper Docker Id when a value is present in the info struct. Is that what the TestConnectPayloadInternalDocker test does?
Yes that's pretty much what the other test does. We can keep this though with your suggestions since this test has a more substantial info struct

Reply:
See c28cb6775c1128750e7bb9b523fd7b1164bfdd52